src/java/org/apache/cassandra/service/accord/async/AsyncOperation.java

```diff
@@ -210,6 +210,7 @@ Iterable<RoutableKey> keys()
 
     private void fail(Throwable throwable)
     {
+        commandStore.agent().onUncaughtException(throwable);
```

Comment:
Mixed feelings as we differ from burn test… knowing is always best but can we see Timeout or Preempt? If so the new logging could spam